wdl/transforms/draft2/src/main/scala/wdl/transforms/draft2/wdlom2wom/WdlDraft2WomExecutableMaker.scala

```diff
@@ -0,0 +1,16 @@
+package wdl.transforms.draft2.wdlom2wom
+
+import common.Checked
+import wdl.WdlNamespaceWithWorkflow
+import wom.executable.{Executable, WomExecutableMaker}
+
+object WdlDraft2WomExecutableMaker extends WomExecutableMaker[WdlNamespaceWithWorkflow] {
+  override def toWomExecutable(wdlNamespaceWithWorkflow: WdlNamespaceWithWorkflow, inputFile: Option[String] = None): Checked[Executable] = {
+    WdlInputParsing.buildWomExecutable(wdlNamespaceWithWorkflow.workflow, inputFile)
+  }
+
+  // TODO Dan: Is adding this back as an implicit class a hideous abuse of power, or the whole point of doing this?
```

Comment:
See my comment regarding simulacrum.  This is done for you but for any A that has an instatnce of  WomExecutableMaker.  Thus you can delete this whole object and for any instance of `WdlNamespaceWithWorkflow` call `.toWomExecutable(inputs)`.  Notice how it moves the `namespace` out of the definition for you because you already have it!

Reply:
I'm going to give this simulacrum a go now (alas poor newly-added implicit classes, I barely knew thee!)